docs/api.rst

```diff
@@ -119,7 +119,7 @@ Inference
 
    hypotest
    test_statistics.qmu
-   utils.loglambdav
+   mle.loglambdav
```

Comment:
For reasons brought up below this should be
```suggestion
   mle.fit
```

and then 
```
mle.fixed_poi_fit
```

should also be added.